text/0685-new-browser-support-policy.md

```diff
@@ -0,0 +1,432 @@
+breaking in the strictest technical sense, it generally is not considered one
+because of the usage patterns that these browsers have in the first place. Their
+users are automatically opted into updating every time they boot the browser,
+which means in almost all cases users update quickly and efficiently. In fact,
+it is often considered a _security issue_ if users are _not_ using the most
+recent version of an evergreen browser.
+
+As such, the fact is that in most cases, if a bug is somehow reported on an
+older version of an unsupported browser, then the fix is usually to tell the
+user to update the browser version, not to patch the application or framework
+for that browser version. In practice, this is how Ember has operated for years,
+and this policy only formalizes this process.
+
+## Implementation
+
+In order to support this policy while keeping all of our tooling on the same
+page, Ember.js itself will add automation to calculate the current minimum
+supported version of browsers on its main branch, and publish them in an
+accessible format. Each time we branch a release, these versions will stop
+updating, so they will effectively be locked in.
+
+These versions will be used for a variety of use cases, such as generating
+documentation and release blog posts (see below), and generating the default
+`config/targets.js` for new Ember apps and addons.
+
+### Implementation timeline
+
+This policy drops support for a major browser, and therefore can only be
+implemented with a major version bump.
+
+## How we teach this
+
+This will be an ongoing process, since the minimum browser versions supported
+change over time. The most important thing here is that users can easily
+determine whether or not a given version of a browser is supported for a given
+Ember release.
+
+The following are the ways we will communicate this:
+
+- For the release blog post for a minor version, we'll include a table which has
+  the list of every supported browser, along with the minimum supported major
+  version of that browser for the release
+- On the [releases page](https://emberjs.com/releases) of the Ember.js website,
+  for each of the listed releases, we will include a table of the supported
+  versions major browsers for that release.
+- The browser support on the releases page for the Stable and LTS branches will
+  be linked in the Ember.js README.
+
+This documentation will be supported by the minimum supported versions that are
+published in the Ember.js package (see above), which will allow them to be
+mostly automated. The supported browser table could look like the following
+(generated using today's usage stats):
+
+### Supported Browsers
+
+#### Desktop
+
+| Chrome | Edge | Firefox  | Safari |
+| ------ | ---- | -------- | ------ |
+| 83     | 18   | 78 (ESR) | 14     |
+
+#### Mobile
+
+| Chrome | Firefox | Safari |
+| ------ | ------- | ------ |
+| 87     | 83      | 14     |
+
+#### Headless
+
+| Chrome | Firefox  |
```

Comment:
I'm confused by references to headless safari in some places and headless firefox is others. Did I miss the explanation somewhere?